Cleaning portrait landscape removed modes

juanky201271 opened the pull request:
I was cleaning a little bit after the decision of remove the IOS Landscape mode. I removed dead code as well, and I removed a dead test as well.

I did this on-top of this PR: https://github.com/zingolabs/zingo-mobile/pull/207

AloeareV commented:
Build is temporarily broken due to zingolib forbidden git shenanigans. 

Once #228 lands (and this is rebased atop it) the build should work again.

juanky201271 commented:
@AloeareV please, can you review this.... Thanks.

juanky201271 commented:
I fixed all the conflicts... @AloeareV please...

fluidvanadium commented:
Hey! I've been working hard on these tests but it didn't get Pulled yet!
several of the changes i made were to logic in the test, for example
typeText was changed to replaceText
the birthday is 1985560 or something

i dont want to try to make these changes in multiple files. my solution is to condense the tests to 2.
i want this to merge and i'll try to rebase

fluidvanadium reviewed (approved):
Hey! I've been working hard on these tests but it didn't get Pulled yet!
several of the changes i made were to logic in the test, for example
typeText was changed to replaceText
the birthday is 1985560 or something

i dont want to try to make these changes in multiple files. my solution is to condense the tests to 2.
i want this to merge and i'll try to rebase

juanky201271 commented on e2e/address_types.test.js:
```diff
@@ -11,10 +11,12 @@ describe('Example', () => {
-    await element(by.id("birthdayinput")).typeText('1985000');
-    await element(by.text("RESTORE WALLET")).tap();
+    await element(by.id('loadingapp.restorewalletseed')).tap();
+    await element(by.id('seed.seedplaceholder')).typeText(
```
Nobody changed your test... as you can see: `typeText`... the same you put here. I did a NEW test, and in MY test I prefer to put `replaceText` because I have no patience to wait the test write the seed.

I don't know if you have any other concerns about this? I'm happy to answer.

fluidvanadium replied:
oh, replaceText is better! im happy for you to change my code when you can improve it

im looking into test fixtures so we can share more code

juanky201271 commented on e2e/new_checkbox_send.test.js:
```diff
@@ -0,0 +1,36 @@
+
+  it('Checkbox send screen -> include UA in memo field', async () => {
+    await element(by.id('loadingapp.restorewalletseed')).tap();
+    await element(by.id('seed.seedplaceholder')).replaceText(
```
This is the test I did, and yes, I prefer: `replaceTest`.

juanky201271 commented:
> Hey! I've been working hard on these tests but it didn't get Pulled yet! several of the changes i made were to logic in the test, for example typeText was changed to replaceText the birthday is 1985560 or something
> 
> i dont want to try to make these changes in multiple files. my solution is to condense the tests to 2. i want this to merge and i'll try to rebase

I think you are talking about another test I added... see this: https://github.com/zingolabs/zingo-mobile/pull/227/files#r1124800018

I didn't change your test, I just reformatting the code & I added some imports... in order to pass the typescript local checks.
